fio.spec

```
@@ -42,74 +59,92 @@ otherwise parameters given to them overriding that setting is given.
 The typical use of fio is to write a job file matching the io load
 one wants to simulate.
 
-%package devel
-Summary:	FIO devel package
-
-%description devel
-FIO devel
-
 %prep
 %setup -q
-find . -type f > source_files.txt
-# fix hard-coded /usr/bin/bash
-sed -i -e "s|/usr/bin/bash|$(command -v bash)|g" tools/genfio
+%{__python3} %{python_pathfix} -i %{__python3} -pn \
+ doc/conf.py \
+ tools/fio_jsonplus_clat2csv \
+ tools/fiologparser.py \
+ tools/hist/*.py \
+ tools/plot/fio2gnuplot \
+ t/*.py
 
 %build
+%if 0%{?suse_version}
+export CFLAGS="%{optflags} -fPIC -pie"
+export CXXFLAGS="%{optflags} -fPIC -pie"
+%endif
 ./configure --disable-optimizations
-EXTFLAGS="$RPM_OPT_FLAGS" make V=1 %{?_smp_mflags}
+env EXTFLAGS="$RPM_OPT_FLAGS" LDFLAGS="$RPM_LD_FLAGS" make V=1 %{?_smp_mflags}
```

FWIW, and no need to push a new commit just for this but the `env` here is superfluous I believe.

```suggestion
EXTFLAGS="$RPM_OPT_FLAGS" LDFLAGS="$RPM_LD_FLAGS" make V=1 %{?_smp_mflags}
```

should work just as well.

---

Remove useless env call

---

> * [x]  Remove useless env call

Fixed with commit c66d57fbc92eb0252d64be9b00b84a52b622300d